Nature 2023 refresh (#53)
springernature/vpat: nature.md

The Nature subscription pages at <a href="https://www.nature.com/nature/subscribe">https://www.nature.com/nature/subscribe</a> currently contain SVG images without text alternatives, and decorative SVG images that are not marked up to be ignored by Assistive Technologies. We are actively working to repair these problems.
<br><br>
Third party scripts may inject non-text content without text alternatives, or with inadequate text alternatives. These include images in advertising scripts and in Disqus commenting widgets. When we become aware of deficiencies, we contact third party suppliers to request repairs.
<br><br>

> When supplied by authors who publish with us, some complex images may lack adequate alternative text, descriptions, and/or captions. Examples may include images of charts, graphs, or maps. Generally, these images are used to supplement text content. A large-scale and currently-active programme of work to improve alt text content submitted by our authors is under way to address this.

Reviewer: > programme of work to improve alt text content

1. Technically, does this programme solely cover `alt` attribute text or does it more widely include the previously mentioned "alternative text, descriptions, and/or captions"?

And if only `alt` text, will the audience understand the meaning of "alt text"? I think it is the first time it is used in this document.

2. `alt` is of course an HTML attribute, but not marked up as such throughout this document unlike e.g. `lang`. Should we do so, or expand to "alternate text" when not explicitly discussing alt attributes?

Author: 1. the latter, the implementation relies on all three features simultaneously. Basically our image descriptions are very complicated and often need to contain HTML markup (e.g. for superscript/subscript, or equations etc), so they can't reasonably go in the alt text. Instead we rely on the visible caption and use the alt text and aria descriptions to associate the figures with the caption material. 

2. Hm, yes, an excellent question, and I agree. I think I'll expand the acronym throughout as we're not always referring to the attribute specifically, even outside of this statement 👍

Author: Oh turns out there were only those 2 instances of it so that wasn't too painful 😅

</ul></td>
<td>Partially Supports</td>
<td>

> Outside of editoral content managed by content management system (CMS), the contents of author-submitted published books and papers, and third party scripts, the pages of this product do not use images of text to convey information except in the case of <a href="https://www.w3.org/WAI/WCAG21/Understanding/images-of-text.html">logotypes</a> (considered essential per WCAG 2.1) or when the image is <a href="https://www.w3.org/TR/WCAG21/#dfn-pure-decoration">pure decoration</a>.

Reviewer: ```suggestion
Outside of editorial content managed by content management system (CMS), the contents of author-submitted published books and papers, and third party scripts, the pages of this product do not use images of text to convey information except in the case of <a href="https://www.w3.org/WAI/WCAG21/Understanding/images-of-text.html">logotypes</a> (considered essential per WCAG 2.1) or when the image is <a href="https://www.w3.org/TR/WCAG21/#dfn-pure-decoration">pure decoration</a>.
```

<br><br>

> CMS content and published books and papers may include complex images that contain images of text. In most cases these images are used to supplement the text content they appear alongside. In some cases (e.g. charts, graphs, or maps, among others), the text content presented alongside may be insufficient to explain the contents of the image. A large-scale and currently-active programme of work to improve alt text content submitted by our authors is under way to address this.

Reviewer: (see previous comment on the use of the phrase "alt text")

Author: Have expanded to "alternative text" in [f973e8b3](https://github.com/springernature/vpat/pull/53/commits/f973e8b3a2470b41d67fd39f6ca58365c3f03cc8)

<td>

> The visual presentation of User Interface Components in this product have a contrast ratio of at least 3:1 against adjacent color(s).

Reviewer: ```suggestion
The visual presentation of User Interface Components in this product have a contrast ratio of at least 3:1 against adjacent colours.
```
We seem to be using British English for our copy this document, and the parenthesis feels... wrong.

<em>Revised Section 508 - does not apply</em>
</td>
<td>Partially Supports</td>

> <td>Additional content that becomes visible on hover or focus (for example tooltips) is extremely rare in this product. In all cases where content is intitally hidden and displayed on interaction, we prefer to use persistent displays that are activated by explicit, unambiguous interactions such as clicks or keypresses.

Reviewer: ```suggestion
<td>Additional content that becomes visible on hover or focus (for example tooltips) is extremely rare in this product. In all cases where content is initially hidden and displayed on interaction, we prefer to apply visibility persistently as a result of unambiguous interactions; such as clicks or keypresses.
```

"persistent displays" felt possibly ambiguous to me, whereas the concept of "visibility" is what is being discussed.

Author: Agree, it felt wrong as I was writing it and hasn't improved after a night's sleep and most of a working day. How do you feel about this as an alternative:

> In all cases where content is intitally hidden and displayed on interaction, we prefer to **make that content persistent until dismissed and** activated by explicit, unambiguous interactions; such as clicks or keypresses.

Author: Reworded after we thrashed this one out in Slack :D

<td>Does Not Support</td>
<td>Language of Parts is not applicable for the majority of this content. All first-party components on the pages of this product that are part of dynamic templates are written in English.
<br><br>
Editorial content or published papers may be written in or include text in languages other than English. Article content may be marked up with a separate <code>lang</code> attribute that matches the written language, though this is not consistently applied.
<br><br>
Third party scripts may inject components with non-English text - for example a targeted advertisement or file presented in a file viewer (e.g. Figshare) may use a different language to the main document. These may not be marked up with a separate <code>lang</code> attribute.
<br><br>

> We currently have no method of confirming the inclusion of appropriate lang attributes for pieces of text in content that may use a different language to the main document.

Reviewer: ```suggestion
We currently have no method of confirming the inclusion of appropriate <code>lang</code> attributes for pieces of text in content that may use a different language to the main document.
```

Consistency.

<tr>
<td>302.3 Without Perception of Color</td>
<td>Supports</td>

> <td>Color perception is not generally needed to use this product.</td>

Reviewer: Do we want to say more about our aims and/or internal practices on this point?

Author: Not here because it's fully described in the WCAG section above, as is the case with all the 508 criteria - the WCAG table is just more granular so it's easier to be explicit about the issues. 

I could copy it over, but it'd just be repetition 😄
